docs/windows.md

```diff
@@ -190,6 +190,8 @@ CPU hotplug is supported. The VM operating system needs to support hotplug and b
 
 RAM hotplug is supported. Note, that while the `pnpmem.sys` driver in use supports RAM hot-remove, the RAM being unplugged has to be not in use and have no reserved pages. In most cases it means, hot-remove won't work. Same as with the CPU hot-remove, when `ch-remote` is invoked to reduce the RAM size, the result will be visible after the OS reboot.
 
+Network device hotplug and hot-remove are supported.
```

Comment:
Disk is also supported (has been from the beginning.)

Reply:
Yeah, I saw that just that we don't test it yet so I thought it would make sense to first mention it after the list from #1916 has been processed.

Thanks